cilium/cmd/service_update.go

```diff
@@ -137,15 +139,26 @@ func updateService(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	if len(backendWeights) == 0 {
+		backendWeights = make([]uint, len(backends))
+		fmt.Printf("Backend weights not set. Setting all weights to 1\n")
```

Comment:
Nit: probably not needed as that's the expected default

Reply:
makes sense, thanks